src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/KeyAgreeRecipientInfoTests.cs

```diff
@@ -14,8 +14,19 @@ namespace System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests
 {
     public static partial class KeyAgreeRecipientInfoTests
     {
-        [Fact]
-        [ActiveIssue(3334, PlatformID.AnyUnix)]
+        [ConditionalFact(nameof(DoesNotSupportKeyAgreementCerts))]
+        public static void TestKeyAgreement_PlatformNotSupportedLinux()
```

Comment:
The "Linux" suffix isn't really correct, since it's all the Unix-family OSes.  "Unix" might be okay, but really I'd leave the suffix off (`TestKeyAgreement_PlatformNotSupported`) and let the conditional fact explain it.